lib/ansible/playbook/role/__init__.py

```diff
@@ -138,8 +140,7 @@ def get_name(self, include_role_fqcn=True):
         return self._role_name
 
     @staticmethod
-    def load(role_include, play, parent_role=None, from_files=None, from_include=False, validate=True):
-
+    def load(role_include, play, parent_role=None, from_files=None, from_include=False, validate=True, public=True):
```

Comment:
Could this have precedence over the _public FA default? I'm wondering if `load`/`__init__` should have public=False. This seems like it could be a breaking change potentially, like here https://github.com/ansible/ansible/blob/stable-2.13/lib/ansible/playbook/role/__init__.py#L426 if public is not passed.

Reply:
not sure, but it should be the default as that is current behaviour of `roles:` and `dependency` .. i do expect that include_role/import_role has precedence over this, but need to test